frontend/src/global_styles/content/work_packages/tabs/_files.sass

```diff
@@ -28,8 +28,11 @@
 
 .op-files-tab,
 .op-attachments-list
+  .op-files-tab--file-upload
+    margin-top: $spot-spacing-0-75
```

Comment:
When I don't have attachments  and no file storage module activated, this results in additional spacing to the tab header which we tried to unify in https://github.com/opf/openproject/pull/10925

<img width="300" alt="Bildschirmfoto 2022-07-20 um 14 49 06" src="https://user-images.githubusercontent.com/7457313/179986213-5bb747ec-6306-4adf-b2cf-5f78ed0401b3.png">

vs.

<img width="300" alt="Bildschirmfoto 2022-07-20 um 14 50 59" src="https://user-images.githubusercontent.com/7457313/179986361-ac6c7b0a-6d6e-4a23-be63-cc0dab9b0a39.png">

Reply:
ah, good find. I still don't get it, why the header is removed here. This would be an easy fix. 

But I can investigate, why the margins do not collapse.